src/index.html

```diff
@@ -8,6 +8,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="page">
+      <div class="page__container">
+        <div class="stars stars--0">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="stars stars--1">
+          <div class="stars__star stars__star--active"></div>
```

Comment:
you don't need this class
<img width="884" alt="image" src="https://github.com/mate-academy/layout_stars/assets/70220667/4decf010-e9a6-4cac-8901-d5f332fdf242">